tutorials/10-configure.md

```diff
@@ -251,3 +251,7 @@ Set Content Security Policy (CSP) headers in express. Default is disabled. When
 #### csrf cookie name
 
 Default is "\_\_csrf". Set to override the cookie name used for a csrf token. This can be helpful to avoid conflicts with other cookies on a domain.
+
+#### exclude non-relevant
+
+When set to `true`, fields with non-relevant values and children of non-relevant parents will be treated as blank in computations and form submissions. Default is `false`.
```

Comment:
Why add "form submissions"? This is the same no matter what, right? I think it would also be helpful to add something like "This matches ODK Collect"